lib/v2/gov/sichuan/deyang/govpulicinfo.js

```diff
@@ -3,112 +3,82 @@ const cheerio = require('cheerio');
 const { parseDate } = require('@/utils/parse-date');
 const { art } = require('@/utils/render');
 const path = require('path');
+const timezone = require('@/utils/timezone');
 
-const rootUrl = 'http://xxgk.deyang.gov.cn/xxgkml2020';
-
-// 地区名称对照表
-const nameDict = {
-    德阳市: 'dys',
-    绵竹市: 'mzs',
-    什邡市: 'sfs',
-    旌阳区: 'jyq',
-    罗江区: 'ljx',
-    广汉市: 'ghs',
-    中江县: 'zjx',
-    高新区: 'gxq',
+// 各地区url信息
+const basicInfoDict = {
+    绵竹市: {
+        rootUrl: 'https://www.mz.gov.cn',
+        infoUrl: 'https://www.mz.gov.cn/info/iList.jsp?tm_id=1805&cur_page=',
+    },
 };
 
-const getInstitutionId = async (ctx, county) => {
-    const url = `${rootUrl}/ptlj.jsp?regionName=${county}`;
-
-    return await ctx.cache.tryGet(`${county}InstitutionId`, async () => {
-        const response = await got.get(url);
+const getInfoUrlList = async (countyName) => {
+    const url = basicInfoDict[countyName].infoUrl;
+    const infoUrlList = [];
+    const expectToday = parseDate(timezone(new Date(), +8));
```

Comment:
Do not add a date if it isn't provided
docs: https://docs.rsshub.app/joinus/pub-date.html